packages/ui/src/composites/wui-input-numeric/index.ts

```diff
@@ -0,0 +1,24 @@
+import { html, LitElement } from 'lit'
+import { customElement, property } from 'lit/decorators.js'
+import { elementStyles, resetStyles } from '../../utils/ThemeUtil'
+
+import styles from './styles'
+
+@customElement('wui-input-numeric')
+export class WuiInputNumeric extends LitElement {
+  public static styles = [resetStyles, elementStyles, styles]
+
+  // -- State & Properties -------------------------------- //
+  @property({ type: Boolean }) public disabled = false
+
+  // -- Render -------------------------------------------- //
+  public render() {
+    return html`<input type="text" inputmode="numeric" ?disabled=${this.disabled} /> `
```

Comment:
```suggestion
    return html`<input type="number" inputmode="numeric" ?disabled=${this.disabled} maxlength="1" /> `
```

In addition to above, ensure to hide default number type input arrow keys added by html (if not done already).